src/components/organisms/AssetContent/index.module.css

```diff
@@ -42,3 +42,7 @@
   margin-left: calc(var(--spacer) / 4);
   margin-right: calc(var(--spacer) / 4);
 }
+
+.content [class*='typeDetails'] {
```

Comment:
just by the way, using attribute selectors to select things should only be a last resort for cases where we do not control markup and can't add a CSS module class. This can pile up to specifity and maintenance problems later on.

So for everything where we control markup a better pattern is to have all components accept an optional `className` so we can set CSS module classes on them wherever we need to style them individually